components/ModSubmissionListContainer.vue

```diff
@@ -16,27 +16,60 @@
             {{ $t("modPanelSubmissionList.submitted") }}
         </div>
 
-        <div v-for="(fakeDatum, index) in modScreenStore.fakeData" :key="index" class="grid grid-cols-subgrid col-span-5 bg-gray-200" >
-            <button @click="handleClickToSubmissionForm(fakeDatum.id)"
-            :data-testid='`mod-submission-list-item-${index + 1}`'
+        <div v-if="hasSubmissions" v-for="(submission, index) in modSubmissionsStore.submissionsData" :key="index"
             class="grid grid-cols-subgrid col-span-5 bg-gray-200">
+            <button @click="handleClickToSubmissionForm(submission.id)"
```

Comment:
is there a reason this is a button vs a clickable div? 
From an accessibility perpective, this might be confusing as a button

Reply:
No reason. I really am still working on semantic HTML